### PR TITLE
166 buffer single point

### LINKED ISSUE #166: Integrate Mocha and Chai libraries with Puppeteer

### PR DESCRIPTION
**Pull request by chriswhong**
- Adds handling for creating bounding box array when there is only a single point in the result set (uses turf buffer and turf bbox to get the bounding box for a 400m buffer around the point)

- Moves `normalized_projects` to a materialized view for better performance

Closes #166

### CODE REVIEW

File: queries/projects/normalized-projects.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+-- this is the base query used for filtering
+
+CREATE MATERIALIZED VIEW normalized_projects AS
```
Comment (allthesignals) [inline queries/projects/normalized-projects.sql:3]: Just making sure I understand: this is a 1-time "migration" query?

File: routes/projects.js [hunk 2]
```diff
@@ -121,6 +124,26 @@ router.get('/', async (req, res) => {
...
+      if (bounds[0][0] === bounds[1][0]) {
+        const point = {
+          type: 'Point',
+          coordinates: [
+            bounds[0][0],
+            bounds[0][1],
+          ],
+        };
+        const buffer = turfBuffer(point, 0.4);
+        console.log(buffer)
```
Comment (allthesignals) [inline routes/projects.js]: Remove plz 👼

**Review by allthesignals — CHANGES_REQUESTED** [review]

File: routes/projects.js (hunk 2 above)
Comment (chriswhong) [reply routes/projects.js]: oops!

File: queries/projects/normalized-projects.sql (hunk 1 above)
Comment (chriswhong) [reply queries/projects/normalized-projects.sql:3]: It needs to be re-run each time the data changes (somehow).  This query is not used by the code, it's just stored here for reference.

File: routes/projects.js (hunk 2 above)
Comment (chriswhong) [reply routes/projects.js]: Fixed

File: queries/projects/normalized-projects.sql (hunk 1 above)
Comment (allthesignals) [reply queries/projects/normalized-projects.sql:3]: Great! That's what I Thought. makes sense, thanks.

**Review by allthesignals — APPROVED** [review]

**Pull request merged**